docs/src/documentation/05-development/04-migration-guides/01-v11.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: v11
+description: How to migrate to Orbit 11.0.0
+redirect_from:
+  - /migration-guides/v11/
+---
+
+# Orbit Migration Guide v11
+
+This migration guide focuses on the process of migrating from Orbit v10.3.0 to v11.0, as some breaking changes were introduced.
```

Comment:
WDYT about encouraging consumers here, mentioning this one is a really simple one besides being a major?

> The best part is, migrating to v11 is incredibly simple! We've streamlined the process to ensure a smooth transition, making it easier than ever to benefit from the latest enhancements.